CesiumGltf/src/FeatureIdTextureView.cpp

```diff
@@ -63,11 +74,19 @@ FeatureIdTextureView::FeatureIdTextureView(
   this->_status = FeatureIdTextureViewStatus::Valid;
 }
 
-int64_t FeatureIdTextureView::getFeatureID(double u, double v) const noexcept {
+int64_t FeatureIdTextureView::getFeatureID(double u, double v) noexcept {
   if (this->_status != FeatureIdTextureViewStatus::Valid) {
     return -1;
   }
 
+  if (this->_pSampler == nullptr) {
+    this->_status = FeatureIdTextureViewStatus::ErrorInvalidSampler;
+    return -1;
+  }
```

Comment:
When `pSampler` is `nullptr`, the status is set to `ErrorUninitialized` or `ErrorInvalidSampler` in the constructors, so I think this check is redundant. The view is meant to validate itself during construction before `getFeatureID` is called, such that `getFeatureID` shouldn't have to change the view's state.

So in other words, I think this check can be removed and `getFeatureID` can remain `const`